.github/workflows/docs.yml

```diff
@@ -0,0 +1,37 @@
+name: Docs
+
+on:
+  #push:
+  #  branches-ignore:
+  #    - '**'
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: macos-latest
+
+    steps:
+    - name: Requirements
+      run: brew install doxygen
```

Comment:
This way, you run a single command, which if it fails provides less clear output. You can split this in multiple commands using `run: |` followed by a newline and a single command for each subsequent line.